webapp/src/components/sidebar_right/confirmation_modal.scss

```diff
@@ -4,7 +4,7 @@
         border-radius: 8px;
         .modal-header {
             border: none;
-            border-bottom: 1px solid #e5e5e5;
+            border-radius: 8px;
```

Comment:
Curious if these tweaks are needed or if the default modal styling just handles everything for us or not? Your screenshot looks like the new default modal as far as I can tell, which is good.

Reply:
Without this radius, the top corners of the modal were square instead of rounded so yeah it was necessary